CMakeLists.txt

```diff
@@ -136,6 +168,11 @@ if(USE_MPI)
   TARGET_LINK_LIBRARIES(_lightgbm ${MPI_CXX_LIBRARIES})
 endif(USE_MPI)
 
+if(USE_OPENMP)
```

Comment:
it seems we didn't need this before. why link to OpenMP_CXX_FLAGS ?

Reply:
I added this when I was getting a seg fault near the code that was doing a parallel loop with OpenMP, but later on I found it was a different issue with my Java wrapper code.  It looks like it is not needed - let me verify and I will remove.

Reply:
verified that it is not needed